spec/requests/evaluations_spec.rb

```diff
@@ -173,4 +173,478 @@
+
+        updated_evaluation = assigns(:evaluation)
+
+        expect(updated_evaluation).to be_persisted
+        expect(updated_evaluation.additional_comments).to eq("Test")
+
+        expect(updated_evaluation.errors).to be_empty
+        expect(updated_evaluation.completed_at).to be_nil
+
+        expect(response).to redirect_to(evaluations_path)
+        expect(flash[:notice]).to include(I18n.t("evaluations.notices.saved_draft"))
+      end
+
+      it "does not allow me to save a draft of an evaluation I did not create", bullet: :skip do
+        user = create(:user, :evaluator)
+        evaluator_submission_assignment = create(:evaluator_submission_assignment, user_id: user.id)
+        evaluation_form = create(:evaluation_form, phase: evaluator_submission_assignment.phase)
+
+        evaluation = create(:evaluation,
+                            user: user,
+                            evaluation_form: evaluation_form,
+                            submission: evaluator_submission_assignment.submission,
+                            evaluator_submission_assignment: evaluator_submission_assignment)
+
+        evaluation_params = { additional_comments: "Test" }
+
+        patch save_draft_evaluation_path(evaluation, params: { evaluation: evaluation_params })
+
+        expect(response).to redirect_to(evaluations_path)
+        expect(flash[:alert]).to eq(I18n.t("evaluations.alerts.unauthorized"))
+      end
+    end
+  end
+
+  # mark_complete_evaluation_path
+  describe "PATCH /evaluations/:id/mark_complete" do
+    context "when logged in as an evaluator" do
+      let(:current_user) { create_user(role: "evaluator") }
+
+      before { log_in_user(current_user) }
+
+      it "allows me to mark my existing evaluation as complete", bullet: :skip do
+        evaluator_submission_assignment = create(:evaluator_submission_assignment, user_id: current_user.id)
+        evaluation_form = create(:evaluation_form, phase: evaluator_submission_assignment.phase)
+
+        evaluation = create(:evaluation,
+                            user: current_user,
+                            evaluation_form: evaluation_form,
+                            submission: evaluator_submission_assignment.submission,
+                            evaluator_submission_assignment: evaluator_submission_assignment)
+
+        evaluation_params = { additional_comments: "Test" }
+
+        patch mark_complete_evaluation_path(evaluation, params: { evaluation: evaluation_params })
+
+        updated_evaluation = assigns(:evaluation)
+
+        expect(updated_evaluation).to be_persisted
+        expect(updated_evaluation.additional_comments).to eq("Test")
+
+        expect(updated_evaluation.errors).to be_empty
+        expect(updated_evaluation.completed_at).not_to be_nil
+
+        expect(response).to redirect_to(evaluations_path)
+        expect(flash[:notice]).to include(I18n.t("evaluations.notices.marked_complete"))
+      end
+
+      # TODO: Needs fix
```

Comment:
what needs fix?

Reply:
I think this was old and I forgot to remove it. When I was having the issue with evaluation_scores failing uniqueness validation